stdlib/SparseArrays/docs/src/AbstractSparseMatrixCSC.md

```diff
@@ -0,0 +1,62 @@
+    AbstractSparseMatrixCSC{Tv,Ti<:Integer} <: AbstractSparseMatrix{Tv,Ti}
+
+Supertype for matrix with compressed sparse column (CSC).
+
+!!! compat "Julia 1.4"
+    `AbstractSparseMatrixCSC` requires at least Julia 1.4.
+
+## `AbstractSparseMatrixCSC` interface
+
+In addition to the [Abstract array interface](@ref man-interface-array), every
+`AbstractSparseMatrixCSC` subtype `TS` must provide the following methods:
+
+* [`size(::TS)`](@ref size)
+* [`getcolptr(::TS) :: AbstractVector{<:Ti}`](@ref getcolptr)
```

Comment:
`getcolptr` is probably going to be renamed to `colptrs` #33041

Reply:
In #25118 @simonbyrne suggested `coloffsets` which I think is a cleaner name.

Reply:
I agree.